### PR TITLE
Refine the colors of some more components

### PR DESCRIPTION
**Pull request by robintown**
This is a reintroduction of https://github.com/matrix-org/matrix-react-sdk/pull/12255, and should be the last part of the [Compound color refinement work](https://github.com/element-hq/element-web/issues/26992). I've updated the following components to reflect [design decisions](https://www.figma.com/file/pqhlirZFEwEp5XpudSbiia/Theming-Web-templates?type=design&node-id=80-60219&mode=design&t=mjqwylUie1vRwzOV-0) that will help them fit in better with the new design system:

- Message bubbles
- Pills
- Sent indicators
- Jump to bottom/unread buttons
- Composer placeholder
- Notification badges
- Room sublists
- Verification shields
- Date separators
- Tooltips

Closes https://github.com/element-hq/element-web/issues/22122

### CODE REVIEW

File: res/css/views/rooms/_E2EIcon.pcss [hunk 1]
```diff
@@ -37,15 +37,6 @@ limitations under the License.
...
 }
 
-/* white infill for the transparency */
-.mx_E2EIcon::before {
-    background-color: #ffffff;
-    mask-image: url("$(res)/img/e2e/normal.svg");
-    mask-repeat: no-repeat;
-    mask-position: center;
-    mask-size: 80%;
-}
```
Comment (t3chguy) [inline res/css/views/rooms/_E2EIcon.pcss:47]: Why can this vanish? Is it intentional the icons become hollow?

Comment (robintown) [reply res/css/views/rooms/_E2EIcon.pcss:47]: That's my interpretation of the designs linked above, that these icons shouldn't use a hard-coded white fill anymore, as it's contrary to how all of our other icons with symbolic cut-outs appear in dark mode

Screenshot from 2024-03-14 15-42-41

Comment (t3chguy) [reply res/css/views/rooms/_E2EIcon.pcss:47]: The design only seems to show the composer tick shield, can we verify with design about the timeline icons and for the other shield variants

Comment (t3chguy) [reply res/css/views/rooms/_E2EIcon.pcss:47]: If the icon came from Compound then it'd be fine but this icon seems to be a legacy one so may want some legacy treatments until it is replaced.

Comment (robintown) [reply res/css/views/rooms/_E2EIcon.pcss:47]: Gaëlle has confirmed that this change is intentional

**Review by t3chguy — APPROVED** [review]

**Review by andybalaam — APPROVED** [review]
Looks good, thanks!

**Pull request merged**